src/main/java/edu/harvard/iq/dataverse/api/Search.java

```diff
@@ -126,6 +126,14 @@ public Response search(
                 geoPoint = getGeoPoint(geoPointRequested);
                 geoRadius = getGeoRadius(geoRadiusRequested);
 
+                if (geoPoint != null && geoRadius == null) {
+                    return error(Response.Status.BAD_REQUEST, "If you supply geo_point you must also supply geo_radius.");
+                }
+
+                if (geoRadius != null && geoPoint == null) {
+                    return error(Response.Status.BAD_REQUEST, "If you supply geo_radius you must also supply geo_point.");
+                }
+
             } catch (Exception ex) {
                 return error(Response.Status.BAD_REQUEST, ex.getLocalizedMessage());
             }
```

Comment:
Do we want to catch a nonnumeric exception here and give more feedback?

Reply:
Well, getGeoPoint and getGeoRadius will already throw exceptions with feedback about non-numeric values.

For example, if the user does this:

    /api/search?q=*&geo_point=41.9580775,-70.6621063&geo_radius=junk

They'll get this error:

    {"status":"ERROR","message":"Non-number radius supplied."}

That said, I may not be following the use cases, the concerns. I'm happy to add more tests for these.